inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/user/UserServiceImpl.java

```diff
@@ -245,4 +262,55 @@ public Boolean delete(Integer userId, String currentUser) {
         return true;
     }
 
+    /**
+     * This implementation is just to simply intercept some requests and reduce the pressure of wrong requests on the
+     * database. The memory method used, the number of multi-node logins is not shared, and a single node is not safe
+     * in high concurrency
+     *
+     * @param req username and pwd
+     */
+    @Override
+    public void login(UserLoginRequest req) {
+        String username = req.getUsername();
+
+        UserLoginLockStatus userLoginLockStatus = loginLockStatusMap.get(username);
```

Comment:
Maybe change to follow is better,
```
UserLoginLockStatus userLoginLockStatus = loginLockStatusMap.getOrDefault(username, new UserLoginLockStatus());
```

Then there is no need for `if` and `else` to check whether the `userLoginLockStatus` is null or not because you check `lockoutTime != null` at the following.